sale_discount_display_amount/report/sale_report_template.xml

```diff
@@ -1,16 +1,107 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
+    <!-- Hiding original Total row from account module -->
+    <template id="hide_original_total_row" inherit_id="account.document_tax_totals">
+        <xpath expr="//tr[contains(@class, 'o_total')]" position="attributes">
+            <attribute name="t-if">False</attribute>
+        </xpath>
+    </template>
+
+    <!-- Custom Total with discount display -->
     <template
         id="sale_discount_display_amount_document"
         inherit_id="sale.report_saleorder_document"
     >
         <xpath expr="//t[@t-call='account.document_tax_totals']" position="after">
+
+            <!-- Add separator line like in original -->
+            <tr class="border-black o_subtotal">
+                <td colspan="2" />
+            </tr>
+
+            <!-- Custom TOTAL line -->
+            <tr>
+                <td><strong>Total</strong></td>
+                <td class="text-end">
+                    <t t-if="doc.discount_total">
+                        <!-- Struck-through total before discount -->
+                        <div
+                            style="text-decoration: line-through; color: #c9302c; font-size: 0.9em;"
```

Comment:
@Mohajiro @ivs-cetmix I'm not sure about this.

This should be done through a class instead (and a css definition too).